iOS/UI/Swipeable Input/DeliveryTypeView.swift

```diff
@@ -11,15 +11,17 @@ import UIKit
 
 class DeliveryTypeView: BaseView {
 
-    let label = ThemeLabel(font: .small)
+    let imageView = UIImageView()
     let button = ThemeButton()
 
     var didSelectContext: ((MessageContext) -> Void)?
 
     override func initializeSubviews() {
         super.initializeSubviews()
 
-        self.addSubview(self.label)
+        self.addSubview(self.imageView)
+        self.imageView.contentMode = .scaleAspectFit
+        self.imageView.tintColor = ThemeColor.T1.color.resolvedColor(with: self.traitCollection)
```

Comment:
These stupid resolved calls are annoying.

Reply:
I don't really understand them but I'm going to let that go for now.